app/assets/templates/service/edit.html.haml

```diff
@@ -1,42 +1,34 @@
-- @angular_form = true
-
-%form.form-horizontal#form_div{"name"          => "angularForm",
-                               "ng-controller" => "serviceFormController",
-                               "ng-show"       => "afterGet"}
-  = render :partial => "layouts/flash_msg"
+%form.form-horizontal#form_div{"name" => "angularForm"}
   %div
     %div
       .form-group{"ng-class" => "{'has-error': angularForm.name.$invalid}"}
-  = render :partial => "layouts/flash_msg"
   %div
     %div
       .form-group{"ng-class" => "{'has-error': angularForm.name.$invalid}"}
-        %label.col-md-2.control-label{"for" => "name"}
+        %label.col-md-2.control-label{"for" => "textInput-markup"}
```

Comment:
@martinpovolny My apologies ... this was done way back in February. Plus it's mostly a copy-paste of the original haml.
Thanks for noticing 👍 -- will be sure to address it.